notebook2/Dockerfile

```diff
@@ -1,29 +1,13 @@
-FROM alpine:3.8
-MAINTAINER Hail Team <hail@broadinstitute.org>
```

Comment:
removed maintainer line, doesn't seem to be used in other project docker files

Reply:
Looks like we're [supposed to use `LABEL` now](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated):
```
LABEL maintainer="hail-team@broadinstitute.org"
```
Not sure it matters since these aren't intended to be publicly shared. On the other hand, it's not much work and it's good practice. I'm in favor of using `LABEL`.